frontend/src/views/web/our-team.html

```diff
@@ -1,124 +1,69 @@
 <div class="white-bg view-container">
     <main-header></main-header>
     <div class="grad-container">
-        <h4 class="team-heading">Team Awesome!</h4>
+        <h4 class="team-heading">{{ourTeam.coreTeamType}}</h4>
         <div class="row">
-            <!-- integrate this card with API -->
-            <div class="col s12 l3 ">
+            <!-- This card is for Core Team -->
+            <div class="col s12 l3" ng-repeat="data in ourTeam.coreTeamDetails">
                 <div class="team-card">
                     <div class="team-link-head">
                         <span>
-    				<a href="http://deshraj.github.io/"><i class="fa fa-linkedin"></i></a>
-    			</span>
+                    <a href="{{data.linkedin_url}}"><i class="fa fa-linkedin"></i></a>
+                </span>
                         <span>
-    				<a href="http://deshraj.github.io/"><i class="fa fa-github"></i></a>
-    			</span>
+                    <a href="{{data.github_url}}"><i class="fa fa-github"></i></a>
+                </span>
                         <span class="right link-team-user">
-    				<a href="http://deshraj.github.io/">deshraj.github.io</a>
-    			</span>
+                    <a href="{{data.personal_website}}">My Profile</a>
+                </span>
                     </div>
                     <div class="team-bg">
-                        <img src="dist/images/deshraj-profile.jpg">
+                        <img src="{{data.background_image}}">
```

Comment:
Also, can you make sure that when there is no backeground image, then you show the similar image as there in http://evalai.cloudcv.org/team contributors list.